### ISSUE TITLE
`aria-label` not in the list of `ComboBoxProps`

### ISSUE THREAD

**Issue by desfero** [adobe/react-spectrum#3373]
# 🐛 Bug Report 

Typescript definitions do not allow passing `aria-label` to `ComboBox` props when visual label is not provided.

## 🤔 Expected Behavior

Given the documentation, 
```
If the combo box does not have a visible label, an aria-label or aria-labelledby prop must be passed instead to identify it to assistive technology.
```
both `aria-label` and `aria-labeledby` should be allowed.
Without that typescript throws an error, if we apply directly `aria-label` into the input we still end up with console warning from `useLabel`
```
If you do not provide a visible label, you must specify an aria-label or aria-labelledby attribute for accessibility 
```

## 😯 Current Behavior

Typescript throws an error 
```
  Object literal may only specify known properties, and ''aria-label'' does not exist in type 'ComboBoxProps<any>'.
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by LFDanLu** [comment]
Definitely an issue, thanks for filing. We must've missed adding `AriaLabelingProps` to the types or something
